test/unit/clickable-component.test.js

```diff
@@ -93,3 +93,21 @@ QUnit.test('handleClick should not be triggered more than once when enabled', fu
   testClickableComponent.dispose();
   player.dispose();
 });
+
+QUnit.test('handleClick should use handler from options', function(assert) {
+  let clicks = 0;
+
+  const player = TestHelpers.makePlayer({});
+  const testClickableComponent = new ClickableComponent(player, {
+    clickHandler() {
+      clicks++;
+    }
+  });
+  const el = testClickableComponent.el();
+
+  Events.trigger(el, 'click');
```

Comment:
can we add a test that if we trigger a `space` or `enter` key on it, the click handler gets called?